contracts/external/traders/LimitOrders.sol

```diff
@@ -59,7 +59,7 @@ contract LimitOrders is
     string constant private EIP712_DOMAIN_NAME = "LimitOrders";
 
     // EIP712 Domain Version value
-    string constant private EIP712_DOMAIN_VERSION = "1.0";
+    string constant private EIP712_DOMAIN_VERSION = "1.1";
```

Comment:
What is this? Didn’t we already deploy this contract?

Reply:
Yeah we deployed it with 1.1.
But I couldn't update the solidity file until now because it would break the tests and message signing